data/WMC/data.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Wrapped MistCoin",
+  "symbol": "WMC",
+  "decimals": "2",
```

Comment:
can you change this to a number. from "2" => 2 @esvictorgc

Reply:
Done! thank you

Reply:
@cfluke-cb can you check if we are OK not to go live? thanks base team!